consistence-coding-standard.md

```diff
@@ -76,6 +76,17 @@ PHP files
 * Long opening tags are used (always `<?php`, never `<?`).
 * There is one empty line after the line with the open tag.
 * File either declares new symbols (classes, functions, constants, etc.) and causes no other side effects, or executes logic with side effects, but should not do both.
+* Uses strict typing by enabling `declare(strict_types = 1);`.
```

Comment:
@lucasvanlierop I want these to be included in Consistence standard, but since Slevomat is using Consistence ruleset, we need to figure out how to set the dependecies, so that it is managable for both sides.